lib/rules/web/admin_console/4.1/common_ui_elements.xyaml

```diff
@@ -50,7 +50,7 @@ check_resource_name_and_icon:
 check_page_contains:
   element:
     selector:
-      xpath: //*[contains(.,'<content>')]
+      visible_text: !ruby/regexp /<content>/i
```

Comment:
Change text matching to `regexp` to avoid failures caused by upper/lower cases

Reply:
Do we really want check content  with case insensitive matching here?

Reply:
I'm afraid yes, sometimes the text changes from `No Events -> No events` while sometimes vice visa. Using case insensitive matching can make sure the whole texts exist no matter upper or lower cases. In fact we don't care too much whether the message are in upper or lower cases

Reply:
Make sense. And How do you think change the rule name to `check_page_match` since it is sort of fuzzy matching?

Reply:
Do we need exact matching for web cases?

Reply:
Ok, let me change it

Reply:
also feature updates see #7021 in cucushift

Reply:
LGTM now. thanks